### PR TITLE
Tu news Saga/webapp 530+webapp 528

### PR DESCRIPTION
**Pull request by orajaby**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

Features covered in this PR:

1. WEBAPP-530: Add view for details about a TüNews.
2. WEBAPP-528: Add TüNews overview.

### CODE REVIEW

File: src/modules/app/route-configs/TuNewsDetailsRouteConfig.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (jafkiitos) [inline src/modules/app/route-configs/TuNewsDetailsRouteConfig.js:1]: @orajaby should not have 2 line breaks on top

File: src/modules/app/route-configs/TuNewsListRouteConfig.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (jafkiitos) [inline src/modules/app/route-configs/TuNewsListRouteConfig.js:1]: Same here

File: src/routes/news/components/LocalNewsDetails.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import styled from 'styled-components'
+import type Moment from 'moment'
+import Page from './../../../modules/common/components/Page'
+
+const StyledContainer = styled.div`
+display: flex;
+flex-direction: column;
+align-items: center;
+`
+
```
Comment (mawadkittos) [inline src/routes/news/components/LocalNewsDetails.js:13]: @orajaby Please remove this line

File: src/routes/news/containers/TuNewsList.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  tuNewsList: TuNewsModel,
+  language: string,
+  city: string,
+  path: string,
+  t: TFunction
+|}
+
+const TU_NEWS = 'tu'
+
+
```
Comment (mawadkittos) [inline src/routes/news/containers/TuNewsList.js:28]: @orajaby No need to this line, please remove it

File: src/routes/news/components/TuNewsElement.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+const Description = styled.div`
+  display: flex;
+  height: 100%;
+  min-width: 1px; /* needed to enable line breaks for too long words, exact value doesn't matter */
+  flex-direction: column;
+  flex-grow: 1;
+  padding: 15px 10px 0;
+  word-wrap: break-word;
+
+
```
Comment (mawadkittos) [inline src/routes/news/components/TuNewsElement.js:38]: No need for this line, please remove it.

**Pull request closed without merging**